query-connector/integration.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status
+
+docker compose -f docker-compose-integration.yaml up -d
+
+# wait for flyway to finish running before...
+docker compose -f docker-compose-integration.yaml logs -f flyway | grep -q "Successfully applied"
+
+# running our integration tests
+DATABASE_URL=postgresql://postgres:pw@localhost:5432/tefca_db TEST_TYPE=integration jest --testPathPattern=tests/integration
```

Comment:
Nice, good use of a script here, would have been a lot to cram into the workflow yaml and allows running tests locally 🔥